app/controllers/api/v1/graphql_controller.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class GraphqlController < ApplicationController
+      def create
+        render json: result
+      rescue StandardError => e
+        raise e unless Rails.env.development?
```

Comment:
@hoangmirs Why do we need to render json error in development, can we not just see the logs and tell the error?

Reply:
This comes from the graphql gem, we can remove it